support/meson/version.sh

```diff
@@ -3,9 +3,14 @@
 # SPDX-FileCopyrightText: 2023 Ledger SAS
 # SPDX-License-Identifier: Apache-2.0
 
-
 if [ "$1" = "get-vcs" ]; then
-  git -C "$MESON_SOURCE_ROOT" describe --tags --dirty
+  GIT_OPTS="--tags --dirty"
```

Comment:
`git describe --tags --dirty --always`  seems to work in the absence of tags. 
And it is described as:
```
       --always
           Show uniquely abbreviated commit object as fallback.
```
Are you sure that we need to check explicitly if tags exist ?